applications/RANSApplication/custom_elements/stabilized_convection_diffusion_reaction_adjoint.h

```diff
@@ -0,0 +1,2477 @@
+
+    ///@}
+    ///@name Operators
+    ///@{
+
+    /// Assignment operator.
+
+    StabilizedConvectionDiffusionReactionAdjointElement& operator=(
+        StabilizedConvectionDiffusionReactionAdjointElement const& rOther)
+    {
+        BaseType::operator=(rOther);
+        Flags::operator=(rOther);
+        // mpProperties = rOther.mpProperties;
+        return *this;
+    }
+
+    ///@}
+    ///@name Informations
+    ///@{
+
+    ///@}
+    ///@name Operations
+    ///@{
+    /**
+     * @brief It creates a new element pointer
+     * @param NewId the ID of the new element
+     * @param ThisNodes the nodes of the new element
+     * @param pProperties the properties assigned to the new element
+     * @return a Pointer to the new element
+     */
+    Element::Pointer Create(IndexType NewId,
+                            NodesArrayType const& ThisNodes,
+                            PropertiesType::Pointer pProperties) const override
+    {
+        KRATOS_TRY;
+        KRATOS_ERROR
+            << "Attempting to Create base "
+               "StabilizedConvectionDiffusionReactionAdjointElement instances."
+            << std::endl;
+        KRATOS_CATCH("");
+    }
+
+    /**
+     * @brief It creates a new element pointer
+     * @param NewId the ID of the new element
+     * @param pGeom the geometry to be employed
+     * @param pProperties the properties assigned to the new element
+     * @return a Pointer to the new element
+     */
+    Element::Pointer Create(IndexType NewId,
+                            GeometryType::Pointer pGeom,
+                            PropertiesType::Pointer pProperties) const override
+    {
+        KRATOS_TRY;
+        KRATOS_ERROR
+            << "Attempting to Create base "
+               "StabilizedConvectionDiffusionReactionAdjointElement instances."
+            << std::endl;
+        KRATOS_CATCH("");
+    }
+
+    /**
+     * @brief It creates a new element pointer and clones the previous element data
+     * @param NewId the ID of the new element
+     * @param ThisNodes the nodes of the new element
+     * @param pProperties the properties assigned to the new element
+     * @return a Pointer to the new element
+     */
+    Element::Pointer Clone(IndexType NewId, NodesArrayType const& ThisNodes) const override
+    {
+        KRATOS_TRY;
+        KRATOS_ERROR
+            << "Attempting to Clone base "
+               "StabilizedConvectionDiffusionReactionAdjointElement instances."
+            << std::endl;
+        KRATOS_CATCH("");
+    }
+
+    /**
+     * this determines the elemental equation ID vector for all elemental
+     * DOFs
+     * @param rResult the elemental equation ID vector
+     * @param rCurrentProcessInfo the current process info instance
+     */
+    void EquationIdVector(EquationIdVectorType& rResult, ProcessInfo& rCurrentProcessInfo) override
+    {
+        std::array<std::size_t, TNumNodes> ids;
+        this->EquationIdArray(ids, rCurrentProcessInfo);
```

Comment:
I don't understand why you need an extra function but I guess you have your reasons